### ISSUE TITLE
Job runs from CLI, but not UI

### ISSUE THREAD

**Issue by fortynachos** [jamesrwhite/minicron#227]
Hi James!

Sorry if this is a duplicate question, but I've read through https://github.com/jamesrwhite/minicron/issues/169 and https://github.com/jamesrwhite/minicron/issues/178 but I'm still stumped.  

0) I installed this using your install script, and am using ruby 2.1.0.
1) Minicron is in my $PATH:
```
$ which minicron
/usr/local/bin/minicron
```
2) I've added the relevant minicron public key to my authorized keys:
```
michael.fotinatos@localhost ~/.ssh
$ cat authorized_keys
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC9LzpeE/rI90yxU9l5JOwExUeHIY+bWOCXgErq0kTJ+Lu3gUUClW7ZqbXq+9bEH9NhxJAzHl92ONXZZczZ9PxnQ1CxvIW5su/FdeMut+Ewkl2y88HMAss4/wHQMTBnaZxGEbhEJpEp3tKD45QkhNvT529Gl+PSuflF35C35y7Rt/uwMjzik2bnUVNWleC9/Ri0DjyV2y8L2aUuTHjcDl/Tk1wUbuyd7pChPDTE9Tc2U/aO3N80/aCUlJsJwec9E11OSNXBImJVb0KBO7yV5S3WdpHMbPGcAs7BmNbwL1UAhwWoo/oQvdF3zfsIio5UvEznnKeTGzZK/P8eBlnOS5dB minicron public key for localhost
```
3) I've tried using the root user when defining the host, but not only does that not work, but it's not really what I want since the commands I want to execute shouldn't use root.

I can kick off jobs using the CLI (```minicron run ls```, for example), but when I go to test the connection on that very same job in the UI, the test fails, and I get a ```Authentication failed for user michael.fotinatos@127.0.0.1```.

Here's my ```/etc/minicron.toml``` in a [gist](https://gist.github.com/fortynachos/55bb6cfd60de0bf60a120dd3eaf552be).  I'm probably just doing something stupid, but could you help point me in the right direction?

**Comment by jamesrwhite** [comment]
Hey!

What OS is this on and are the client and hub and on the same server?

Can you try manually connecting like so and see what it says?

`ssh -v -i ~/.ssh/minicron_host_X_rsa user@localhost`

Where `X` is the ID of the host that you see in the URL when you view it and `user` is the user you configured the host to use for ssh.

**Comment by jamesrwhite** [comment]
I'm closing this due to inactivity, feel free to re-open if you still wish to discuss it further.

**Comment by yizhexu** [comment]
Hi! I ran into similar problem. authorized key set up similarly to @jamesrwhite 
Running ssh -v -i ..ssh/minicron_host_x_rsa yizhe@silver gives me the following message:
ssh -v -i .ssh/minicron_host_2_rsa yizhe@silver
OpenSSH_6.6.1, OpenSSL 1.0.1e-fips 11 Feb 2013
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 56: Applying options for *
debug1: Connecting to silver [fe80::d250:99ff:fe86:3cf0%enp0s31f6] port 22.
debug1: Connection established.
debug1: identity file .ssh/minicron_host_2_rsa type 1
debug1: identity file .ssh/minicron_host_2_rsa-cert type -1
debug1: Enabling compatibility mode for protocol 2.0
debug1: Local version string SSH-2.0-OpenSSH_6.6.1
debug1: Remote protocol version 2.0, remote software version OpenSSH_6.6.1
debug1: match: OpenSSH_6.6.1 pat OpenSSH_6.6.1* compat 0x04000000
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: server->client aes128-ctr hmac-md5-etm@openssh.com none
debug1: kex: client->server aes128-ctr hmac-md5-etm@openssh.com none
debug1: kex: curve25519-sha256@libssh.org need=16 dh_need=16
debug1: kex: curve25519-sha256@libssh.org need=16 dh_need=16
debug1: sending SSH2_MSG_KEX_ECDH_INIT
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug1: Server host key: RSA 41:01:92:57:df:3c:ed:64:44:7b:e4:f6:4d:74:1e:b4
debug1: Host 'silver' is known and matches the RSA host key.
debug1: Found key in /home/yizhe/.ssh/known_hosts:1
debug1: ssh_rsa_verify: signature correct
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: SSH2_MSG_NEWKEYS received
debug1: SSH2_MSG_SERVICE_REQUEST sent
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: publickey,gssapi-keyex,gssapi-with-mic,password
debug1: Next authentication method: gssapi-keyex
debug1: No valid Key exchange context
debug1: Next authentication method: gssapi-with-mic
debug1: Unspecified GSS failure.  Minor code may provide more information
No Kerberos credentials available

debug1: Unspecified GSS failure.  Minor code may provide more information
No Kerberos credentials available

debug1: Unspecified GSS failure.  Minor code may provide more information


debug1: Unspecified GSS failure.  Minor code may provide more information
No Kerberos credentials available

debug1: Next authentication method: publickey
debug1: Offering RSA public key: .ssh/minicron_host_2_rsa
debug1: Authentications that can continue: publickey,gssapi-keyex,gssapi-with-mic,password
debug1: Next authentication method: password
yizhe@silver's password:
